src/systems/filecoin_vm/runtime/runtime.go

```diff
@@ -429,7 +433,7 @@ func (rtOuter *VMContext) _sendInternal(input InvocInput, errSpec ErrorHandlingS
 		rtOuter._toplevelSender,
 		rtOuter._toplevelBlockWinner,
 		rtOuter._toplevelSenderCallSeqNum,
-		rtOuter._internalCallSeqNum+1,
+		rtOuter._internalCallSeqNum,
```

Comment:
[Blocking] Why was the +1 removed? The internal CallSeqNum needs to be incremented somewhere, independently from the toplevel CallSeqNum.

Reply:
Sorry, I misread. Reverted.